src/ethereum/api/fragments.ts

```diff
@@ -54,7 +54,6 @@ export const thirdPartyFragment = () => gql`
     root
     managers
     maxItems
-    totalItems
```

Comment:
Shouldn't we remove it from ThirdPartyFragment too?

Reply:
We do! Great catch!